next-types.d.ts

```diff
@@ -336,3 +336,12 @@ interface DbUserList {
   readonly created_at: string;
   readonly updated_at: string;
 }
+
+interface DBList {
```

Comment:
Is this what the list should be called or do we have another name for it? For example, the user list is `DBUserList`.	It seems to be a generic list for stuff being pulled from `/lists/${id}` in the API.